TestCoins.c

```diff
@@ -0,0 +1,95 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// Ayisha Sameera,GET.
+// ------------------------------------------------------------------
+// TestCoins.c
+// Program on branch Test 2.2
+// Program that tests Coins.c
+// ------------------------------------------------------------------------------------------------
+
+#include <stdio.h>
+#include <stdbool.h>
+#include <ctype.h>
+#include <string.h>
+#include <stdlib.h>
+#include "Coins.h"
+
+static bool isEqual (Coins a, Coins b) {
+   return a.Ten == b.Ten && a.Five == b.Five && a.Two == b.Two && a.One == b.One;
+}
+
+static bool Choice () {
+   while (1) {
+      switch (tolower (getchar ())) {
+         case 'y': while (getchar () != '\n'); return true;
+         case 'n': while (getchar () != '\n'); return false;
+         case '\n': break;
+         default:  while (getchar () != '\n'); break;
+      }
+      printf ("Please enter (y/n) : ");
+   }
+
+}
+
+static void Test_Change () {
+   int balance[] = { 1,10,5,2,23456,76456,99999,968 };
+   Coins expected[] = { { 0,0,0,1 },{ 1,0,0,0 },{ 0,1,0,0 },{ 0,0,1,0 },
+      { 2345,1,0,1 },{ 7645,1,0,1 }, { 9999,1,2,0 },{ 96,1,1,1 } };
+   int length = sizeof (balance) / sizeof (balance[0]);
+   printf ("--------------------------------------------------\n");
+   printf ("|     Balance            |        Result         |\n");
+   printf ("--------------------------------------------------\n");
+   for (int i = 0; i < length; i++) {
+      Coins actual = Change (balance[i]);
+      printf ("|    %6d              |        %5s          |\n", balance[i], isEqual (actual, expected[i]) ? "Pass" : "Fail");
+      printf ("--------------------------------------------------\n");
+   }
+

```

Comment:
1 extra line is left _after_ the function is over so everything doesn't appear too cluttered. Or to improve readability in complex pieces of code. 
```suggestion
```